db/schema.rb

```diff
@@ -63,13 +69,16 @@
     t.integer "quantity"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
```

Comment:
Not sure if you did rails db:migrate but it looks like photo_url, description, and status are missing because of the typo on def change.

Reply:
Thank you for catching that! I will re-run it and generate a new pull request